labs/src/main/scala/org/scalalabs/basic/lab02/ListManipulationExercise01.scala

```diff
@@ -88,7 +79,7 @@ object ListManipulationExercise01 {
    * pattern match or some other method.
    */
   def oddElements(iList: List[Int]): List[Int] = {
-    error("fix me")
+    iList.zipWithIndex.collect{ case (e, i) if ((i + 1) % 2) != 0 => e }
```

Comment:
Oh yeah, I noticed I had an off by one error so I solved it pretty much the worst way possible.  That interpretation didn't even occur to me - and the test isn't particularly helpful either.

@bwiedermann what is the intent here?